lib/data_pull.rb

```diff
@@ -46,12 +49,16 @@ def run
       return
     end
 
-    result = subtask_class.new.run(args: argv, include_missing: config.include_missing?)
+    result = subtask_class.new.run(args: argv, config:)
```

Comment:
cc @olatifflexion this is definitely going to cause merge conflicts with your branch to add a new task, sorry! I think it should be pretty stable now